### PR TITLE
Fix several typos in README.md

### PR DESCRIPTION
**Pull request by tisnik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ See: https://github.com/fabric8io/fabric8-test/blob/master/ee_tests/README.md
 
 ## API tests on OSiO
 
-Our API tests use [pyresttest](https://github.com/svanoort/pyresttest.git) a pyton test framework that make all your API test declarative. No need to learn python, your test are plain old yaml file!
+Our API tests use [pyresttest](https://github.com/svanoort/pyresttest.git) a Python test framework that make all your API test declarative. No need to learn python, your test are plain old yaml file!
```
Comment (ljelinkova) [inline README.md]: Python test framework that make**s** 
all your API test**s**
no need to learn **P**ython
your test**s**

File: README.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ cd EE_API_automation/pyresttest/
 ./run_forge_import_api_test.sh YOUR_OSIO_ACOUNT YOUR_OSIO_TOKEN
 ```
 
-> NOTE: Depending wether your github account belongto an organisation or not the API response is different. the bash script wraps the differentce and run the relevant tests.
+> NOTE: Depending whether your GitHub account belong to an organisation or not the API response is different. the bash script wraps the difference and run the relevant tests.
```
Comment (ljelinkova) [inline README.md]: belong**s** to
**T**he bash 
run**s** the

File: README.md (hunk 1 above)
Comment (tisnik) [reply README.md]: nice, TY

File: README.md (hunk 2 above)
Comment (tisnik) [reply README.md]: again, thank you!

**Pull request merged**